Stripe/STPCardIOProxy.m

```diff
@@ -0,0 +1,129 @@
+//
+//  STPCardIOProxy.m
+//  Stripe
+//
+//  Created by Ben Guo on 5/12/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import "STPCardIOProxy.h"
+#import "STPCardParams.h"
+#import "STPAnalyticsClient.h"
+
+@protocol STPClassProxy
```

Comment:
Hmm – I think I still prefer this approach. It feels more lightweight not to add a bunch of external headers, and I like that these proxy classes explicitly only define the selectors that we use. I also like that this approach guards against our SDK trying to call into an incompatible version of card.io, by checking that each selector we call is present. We could still do that if we copied the headers, but I like that this way feels more explicit.